packages/components/src/utils/config-values.js

```diff
@@ -47,6 +47,7 @@ export default Object.assign( {}, CONTROL_PROPS, TOGGLE_GROUP_CONTROL_PROPS, {
 	borderWidthFocus: '1.5px',
 	borderWidthTab: '4px',
 	spinnerSize: 16,
+	progressBarWidth: 160,
```

Comment:
Two potential confusion points with naming here:

- Above, I mentioned the one that might be confusing because of `width` vs `max-width`. This also applies here
- We don't really use this for width, but rather for `default` width. That being said, this config value will be overridden by the prop too, so it might be useful to make it clear that it's a default (max) width.

Reply:
> Above, I mentioned the one that might be confusing because of width vs max-width. This also applies here
👍🏻 

> We don't really use this for width, but rather for default width. That being said, this config value will be overridden by the prop too, so it might be useful to make it clear that it's a default (max) width.

Cool, I'll rename it accordingly.